AEPIdentity/Sources/IdentityState.swift

```diff
@@ -252,6 +251,37 @@ class IdentityState {
         lastValidConfig = newConfig
     }
 
+    /// Invoke each time when we receive Analytics Response event
+    /// - Parameters:
+    ///   - event: the event from Analytics Respsonse
+    ///   - eventDispatcher: a function which when invoked dispatches an `Event` to the `EventHub`
+    func handleAnalyticsResponse(event: Event, eventDispatcher: (Event) -> Void) {
+        guard let aid = event.data?[IdentityConstants.Analytics.ANALYTICS_ID] as? String, !aid.isEmpty else {
```

Comment:
Add reading the `aid` as we do for other event data inside of `Event+Identity`. Then use `event.aid`.

Reply:
updated